.travis.yml

```diff
@@ -2,4 +2,4 @@ language: node_js
 node_js:
   - 12
 script:
-  - yarn lint 
+  - yarn test
```

Comment:
Maybe you can write `yarn lint && yarn test`, or run `yarn lint`  as `beforeScript:` option

Reply:
https://github.com/crewaii/protocol/blob/f353c434ceca7147e711073222adc3a3ac6b7fcd/package.json#L12 `test` command includes `yarn lint`